sdk/execution/execution.go

```diff
@@ -22,18 +20,18 @@ const (
 // Execution exposes execution APIs of MESG.
 type Execution struct {
 	ps     *pubsub.PubSub
-	m      manager.Manager
 	db     database.ServiceDB
 	execDB database.ExecutionDB
+	instDB database.InstanceDB
 }
 
 // New creates a new Execution SDK with given options.
-func New(m manager.Manager, ps *pubsub.PubSub, db database.ServiceDB, execDB database.ExecutionDB) *Execution {
+func New(ps *pubsub.PubSub, db database.ServiceDB, execDB database.ExecutionDB, instDB database.InstanceDB) *Execution {
 	return &Execution{
-		m:      m,
 		ps:     ps,
 		db:     db,
```

Comment:
The service DB should not be injected as a dependency, but the ServiceSDK should